<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="LAPTOP-PE7CMH67" name="com.iPresence.BookingTest" tests="10" failures="1" timestamp="21 Nov 2019 00:22:16 GMT" time="17.127" errors="0">
  <testcase name="TestEmptySecurityCode" time="0.739" classname="com.iPresence.BookingTest"/>
  <testcase name="TestLinksValidation" time="10.565" classname="com.iPresence.BookingTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at org.testng.Assert.assertTrue(Assert.java:51)
	at com.iPresence.BookingTest.TestLinksValidation(BookingTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1297)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TestLinksValidation -->
  <testcase name="TestEmptyEmail" time="0.577" classname="com.iPresence.BookingTest"/>
  <testcase name="TestGuestLoginButton" time="0.634" classname="com.iPresence.BookingTest"/>
  <testcase name="TestEmptyFirstName" time="0.519" classname="com.iPresence.BookingTest"/>
  <testcase name="TestEmptyLastName" time="0.715" classname="com.iPresence.BookingTest"/>
  <testcase name="TestBookingLoginButton" time="1.162" classname="com.iPresence.BookingTest"/>
  <testcase name="TestEmptyPhone" time="0.537" classname="com.iPresence.BookingTest"/>
  <testcase name="TestEmptyCardHolderName" time="0.799" classname="com.iPresence.BookingTest"/>
  <testcase name="TestEmptyCardNumber" time="0.880" classname="com.iPresence.BookingTest"/>
</testsuite> <!-- com.iPresence.BookingTest -->
